diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
index 5f63978..b313af2 100644
--- a/node_modules/node-pty/binding.gyp
+++ b/node_modules/node-pty/binding.gyp
@@ -5,9 +5,6 @@
     ],
     'conditions': [
       ['OS=="win"', {
-        'msvs_configuration_attributes': {
-          'SpectreMitigation': 'Spectre'
-        },
         'msvs_settings': {
             'VCCLCompilerTool': {
               'AdditionalOptions': [
@@ -48,26 +45,6 @@
             'src/win/conpty_console_list.cc'
           ],
         },
-        {
-          'target_name': 'pty',
-          'include_dirs' : [
-            '<!(node -p "require(\'node-addon-api\').include_dir")',
-            'deps/winpty/src/include',
-          ],
-          # Disabled due to winpty
-          'msvs_disabled_warnings': [ 4506, 4530 ],
-          'dependencies' : [
-            'deps/winpty/src/winpty.gyp:winpty-agent',
-            'deps/winpty/src/winpty.gyp:winpty',
-          ],
-          'sources' : [
-            'src/win/winpty.cc',
-            'src/win/path_util.cc'
-          ],
-          'libraries': [
-            '-lshlwapi'
-          ],
-        }
       ]
     }, { # OS!="win"
       'targets': [
